### ISSUE TITLE
Deprecation warnings about `PluginCustomValidator` usage by this app

### ISSUE THREAD

**Issue by glennmatthews** [nautobot/nautobot-app-data-validation-engine#181]
### Environment
* Python version:  
* Nautobot version:  2.3.5
* nautobot-data-validation-engine version: 3.2.0

### Expected Behavior

Nautobot to start up without deprecation warnings

### Observed Behavior

With `NAUTOBOT_LOG_DEPRECATION_WARNINGS` enabled, one warning is emitted per model supporting custom validators, e.g.:

```
19:37:07.951 WARNING nautobot.core.utils.deprecation deprecation.py                   init_subclass() :
  Class PluginCustomValidator is deprecated, and will be removed in a future Nautobot release. Instead of deriving DcimPowerporttemplateCustomValidator from PluginCustomValidator, please migrate your code to inherit from class CustomValidator instead.
```

It looks like this warning was added in Nautobot v1.5.2 by nautobot/nautobot#2877 so it should be safe to switch to using `CustomValidator` under all current Nautobot versions.

### Steps to Reproduce
1.
2.
3.

**Comment by nrnvgh** [comment]
No big deal; making a cleanup pass at my tickets and saw this was open. Pinging to make sure it's on the radar is all.
